cluster/saltbase/salt/kubelet/default

```diff
@@ -151,6 +151,8 @@
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% elif pillar.get('network_provider', '').lower() == 'cni' %}
   {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+{%elif pillar.get('policy_provider', '').lower() == 'calico' and grains['roles'][0] != 'kubernetes-master' -%}
```

Comment:
Well, it's not going to happen for 1.3 for sure, and it's a pretty high barrier to adoption in general, IMO.  I'm not sure I understand why that is needed for policy.